### DISCUSSION TITLE
Color distortion issue with PDFs (especially visible in green)

### DISCUSSION THREAD

**Discussion by jjojala** [OpenOrienteering/mapper#2072] [Questions & Answers] [answered]
I've noticed, that when exporting PDF or printing directly from OOM,  colors are bright and I believe correct (though not verified toward IOF's color reference). But when you print OOM exported PDF to paper (common convention) or "print" to PDF file (I used PDF FIll's driver) You'll have copy having distorted colors. Especially distortion is visible on green color, but one can see distortion easily also in blue.

As the convention of distribute maps as PDF's is pretty common, this color distortion issue is annoying. Is this purely because of printer specific color profiles and what can one do for this?

I also tried "Microsoft Print to PDF" -driver, which produced non-distored colors (?)

On the right, sample from the PDF exported from OOM, and on the left the result when printing PDF (PDF Fill's driver).
![color-sample](https://user-images.githubusercontent.com/6313950/171817403-6d333fe2-7024-43e3-8016-acb03f8c0be9.png)

**Comment by dl3sdo** [comment, marked as answer]
Hello @jjojala, did you chose 'Device CMYK' as Color mode for the sample on the left side? This mode is selectable for PDF exports in 'Vector graphics' mode (not available in 0.95 but in 0.94).
I use 0.94 to create PDFs by selecting CMYK for giving the PDF to copy shops.

**Reply by jjojala** [reply]
Nope, that was created from Adobe Acrobat Reader by printing to PDF -file by using PDF Fill driver. But the original (on the right side) was exported to PDF from OOM by using "default" color mode. Obviously, Adobe Acrobat Reader or PDF Fill driver uses something similar to "Device CMYK". 

But now when you mentioned that color mode, I tried "Device CMYK" color mode to export PDF and managed to get something similar as in the left sample. Haven't really noticed that setting until now. Thanks, this was helpful.
